src/cloudant/_common_util.py

```diff
@@ -287,66 +290,187 @@ def __new__(cls, code):
             return str.__new__(cls, code.encode('utf8'))
         return str.__new__(cls, code)
 
-class InfiniteSession(Session):
+
+class ClientSession(Session):
```

Comment:
It's a little tricky as `ClientSession` is used directly when admin party is enabled.
I still think it's worth doing. I've raise issue #310 to track the work.